### ISSUE TITLE
Have the ability to save and recall an event sonification scheme

### ISSUE THREAD

**Issue by tpmccauley** [KonVas/Ipsosboard#4]
Breaking off number 3 from #2

**Comment by tpmccauley** [comment]
This can be very ambitious or modest. 

Ambitious means saving the whole state including complete event information, what's checked (and corresponding values), and the corresponding synths. (This may be useful anyway for logging purposes)

However for now it may be good-enough (and advisable given that the interface is in-flux) to just save the current event index and the information on which buttons are checked. This is sufficient to recreate the state (which is re-triggered by the reactive state of the activeEventName).

**Comment by tpmccauley** [comment]
Need to save state of sliders as well?

**Comment by muellmusik** [comment]
Yes. I’ll leave ambitious or modest to you, bearing in mind that we need this to work a week Tuesday ;-)
